core/src/main/scala/org/apache/spark/mapred/SparkHadoopMapRedUtil.scala

```diff
@@ -65,3 +71,77 @@ trait SparkHadoopMapRedUtil {
     }
   }
 }
+
+object SparkHadoopMapRedUtil extends Logging {
+  def commitTask(
```

Comment:
nit: maybe place this overloaded version below the main one as it is merely a convenience method (also allows us to avoid duplicating the documentation)